src/data/content/homepage.ts

```diff
@@ -9,78 +9,46 @@ export default {
     image: 'https://files.ably.io/website/images/meta-tags/ably-generic%402x.jpg',
     twitter: '@ablyrealtime',
   },
-  sections: [
-    {
-      columns: 1,
-      bottomMargin: 48,
-      cards: [
-        {
-          title: 'Ably Documentation',
-          type: 'hero',
-          content: 'Realtime experience infrastructure that just works at any scale.',
-        },
-      ],
+  platformProducts: {
+    title: 'Platform & Products',
+    platformCard: {
+      title: 'Platform',
+      content: "Understand the core concepts of Ably's platform.",
```

Comment:
I think ideally we shouldn't couple the descriptions for the product assets here with those set in the meganav. It also looks odd to then manage 'platform' separately, but can we have an override for the descriptions in docs?

Reply:
@m-hulbert the ablyui component doesn't have a way to change that, i'll have to update it first, is this something you'd prefer before releasing this?

Reply:
@m-hulbert i'll mark this as resolved, when you get an update from marketing, we can take it from there